### PR TITLE
README Clarification

### PR DESCRIPTION
**Pull request by DaviDeil**
I mistook the web browser's developer console to mean the server's. Because below in the FAQ it was mentioned on how to tell that it's up by pinging it, so that was the only context I had for using these commands in a developer console.

This might help other's with the same mindset as me.

### CODE REVIEW

**Comment by MrPrimate** [comment]
People might make the same mistake here, if they are hosting Foundry, can you reword it to include both,.

**Comment by DaviDeil** [comment]
I updated it again, but I'm unsure if I used the right language... hm.
